### PR TITLE
Update temp.py

### PR DESCRIPTION
**Pull request by jaykar-devzone**
Task:

### CODE REVIEW

File: temp.py [hunk 1]
```diff
@@ -1 +1,3 @@
 # This is Temp File
+
+# This is python file . pls chnag to js
```
Comment (abrarmalekji12) [inline temp.py:3]: please keep it same, and add print function

**Review by abrarmalekji12 — CHANGES_REQUESTED** [review]
please apply changes

File: temp.py (hunk 1 above)
Comment (jaykar-devzone) [reply temp.py:3]: done

**Review by jaykar-devzone — COMMENTED** [review]
I have done

**Review by abrarmalekji12 — APPROVED** [review]

**Pull request merged**